test/kafka_test.go

```diff
@@ -186,6 +186,17 @@ func (s *CLITestSuite) TestKafkaClusterConfiguration() {
 		test.workflow = true
 		s.runIntegrationTest(test)
 	}
+
+	kafkaRestURL := s.TestBackend.GetKafkaRestUrl()
+	tests = []CLITest{
+		{args: "kafka cluster configuration update --config compression.type=zip,sasl_mechanism=SASL/PLAIN", fixture: "kafka/cluster/configuration/update-onprem.golden"},
+	}
+
+	for _, test := range tests {
+		test.login = "onprem"
+		test.env = []string{"CONFLUENT_REST_URL=" + kafkaRestURL}
+		s.runIntegrationTest(test)
+	}
```

Comment:
Doesn't look like we need a `for` loop here

Reply:
Ah, I left that in because I'm planning to add a companion `kafka cluster configuration list` command for on-prem, and hence more tests here.